src/components/activity-history-list/activity-record-item.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { Tr, Th, Td } from '@mtfh/common';
+
+export interface ActivityRecordItemProp {
```

Comment:
`ActivityRecordItemProps`

Reply:
updated `ActivityRecordItemProp` to `ActivityRecordItemProps`